.github/workflows/ci.yaml

```diff
@@ -87,6 +87,30 @@ jobs:
         run: |
           bash ./ci/ensure_formated.sh
 
+  bazel:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          ssh-key: ${{ secrets.SSH_KEY }}
+
+      - name: Install clang-14
```

Comment:
I think we can just use the system CC for now.

I'm going to punt on requiring to use clang as our compiler until I see if my bazel PR can get merged.